### PR TITLE
aws integration: implement connect ui

### PR DESCRIPTION
**Pull request by itayd**
also remove unused files from web

### CODE REVIEW

File: integrations/aws/save.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	// Read and parse POST request body.
+	err := r.ParseForm()
+	if err != nil {
+		http.Error(w, "Form parsing error: "+err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	initData := sdktypes.EncodeVars(authData{
+		Region:      strings.TrimSpace(r.FormValue("region")),
+		AccessKeyID: strings.TrimSpace(r.FormValue("access_key")),
```
Comment (efiShtain) [inline integrations/aws/save.go:46]: you can't use this parameters as generic vars, 
they have to be stored as secrets

**Review by efiShtain — CHANGES_REQUESTED** [review]
Please store secret values as secrets

File: integrations/aws/save.go (hunk 1 above)
Comment (itayd) [reply integrations/aws/save.go:46]: they are - see field tags on `authData`

Comment (itayd) [reply integrations/aws/save.go:46]: image

File: integrations/aws/save.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		SecretKey:   strings.TrimSpace(r.FormValue("secret_key")),
+		Token:       strings.TrimSpace(r.FormValue("token")),
+	})
+
+	sdkintegrations.FinalizeConnectionInit(w, r, integrationID, initData)
```
Comment (daabr) [inline integrations/aws/save.go:51]: Is there a test method in the AWS API to check the validity of the supplied details?

File: web/static/aws/connect/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (daabr) [inline web/static/aws/connect/index.html:1]: Your HTML isn't auto-formatted - please use Prettier as your HTML formatter in VS Code.

File: web/static/aws/connect/index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+  <table class="title">
+    <td><img class="logo" src="/static/images/aws.svg" alt="Logo" /></td>
+    <td>
+      <h1>AWS - Create a New Connection</h1>
+    </td>
+  </table>
+
+
+  <form id="save-form" method="post" action="/aws/save">
```
Comment (daabr) [inline web/static/aws/connect/index.html]:
```suggestion
  <!-- TODO(ENG-799) -->
  <form id="save-form" method="post" action="/aws/save">
```

File: web/static/aws/connect/index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="form-group">
+      <label for="region"> Region </label>
```
Comment (daabr) [inline web/static/aws/connect/index.html]: In all `<label>` elements here - there's no need for spaces around the text.

File: web/static/aws/connect/index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+      <input type="text" name="region" autocomplete="off" spellcheck="false" class="text" required />
```
Comment (daabr) [inline web/static/aws/connect/index.html]: Should this be a drop-down selection instead of free-form text?

File: web/static/chatgpt/connect/success.html
Comment (daabr) [inline web/static/chatgpt/connect/success.html:1]: Please do all the `success.*` file deletions in a separate PR. This is unrelated to AWS.

File: web/static/embed.go [hunk 7]
```diff
@@ -42,3 +42,6 @@ var SlackWebContent embed.FS
 
 //go:embed twilio/connect
 var TwilioWebContent embed.FS
+
+//go:embed aws/connect
+var AWSWebContent embed.FS
```
Comment (daabr) [inline web/static/embed.go]: Please move this blocks above ChatGPT, not at the bottom of the file

**Review by daabr — DISMISSED** [review]

**Review by efiShtain — APPROVED** [review]

File: web/static/aws/connect/index.html (hunk 5 above)
Comment (itayd) [reply web/static/aws/connect/index.html]: done

File: web/static/chatgpt/connect/success.html
Comment (itayd) [reply web/static/chatgpt/connect/success.html:1]: separated

File: web/static/embed.go (hunk 7 above)
Comment (itayd) [reply web/static/embed.go]: done

File: web/static/aws/connect/index.html (hunk 4 above)
Comment (itayd) [reply web/static/aws/connect/index.html]: done

File: web/static/aws/connect/index.html (hunk 3 above)
Comment (itayd) [reply web/static/aws/connect/index.html:1]: it is according to my formatter

File: web/static/aws/connect/index.html (hunk 6 above)
Comment (itayd) [reply web/static/aws/connect/index.html]: done, combined

File: integrations/aws/save.go (hunk 2 above)
Comment (itayd) [reply integrations/aws/save.go:51]: not that i've seen :/

File: web/static/chatgpt/connect/success.html
Comment (itayd) [reply web/static/chatgpt/connect/success.html:1]: https://github.com/autokitteh/autokitteh/pull/275
